tapalcatl_server/handler.go

```diff
@@ -116,6 +121,10 @@ func logBool(x bool) string {
 	}
 }
 
+func logDuration(x time.Duration) string {
+	return fmt.Sprintf("%d", x.Nanoseconds())
```

Comment:
Are we using a time of zero to mean something special?

If not, I'd emit the zero times as well, so that the averages and histograms will be accurate, should we be lucky enough to have things which take <1000μs :smile: